src/CreateRecordsWrapper.js

```diff
@@ -43,10 +43,35 @@ const CreateRecordsWrapper = ({
     createItemRecord,
   },
 }) => {
-  const { identifierTypesByName } = useData();
+  const { identifierTypesByName, instanceStatuses, settings } = useData();
   const callout = useCallout();
   const isLoading = useIsLoading();
 
+  const value = settings[0]?.value || '';
+
+  let config;
+
+  try {
+    const { instanceStatusCode, defaultDiscoverySuppress } = JSON.parse(value);
```

Comment:
@SergiyVSergiyenko I wonder if we could move this parsing / cleaning up to https://github.com/folio-org/ui-plugin-create-inventory-records/blob/master/src/providers/DataProvider.js#L25. This way it will only happen once and not on every `CreateRecordsWrapper` render.  `CreateRecordsWrapper` would also get exactly what it expects. What do you think about that?

Reply:
Great idea! And now it looks elegant. 😄